packages/mobile/src/helpers/helpers.ts

```diff
@@ -1 +1,2 @@
 export { getQueryString } from './http/http';
+export * from './image/image';
```

Comment:
Let's be explicit here
```suggestion
export { pickImageLibrary, pickImageCamera } from './image/image';
```

Reply:
updated